lessons/beginners/install-editor/vscode.md

```diff
@@ -0,0 +1,29 @@
+{% set editor_name = 'VSCODE' %} {% set editor_url = 'https://code.visualstudio.com' %} 
+{% extends lesson.slug + '/_base.md' %}
+
+{% block name_gen %} Instalace VSCode{% endblock %}
+{% block setup %}
+
+Editor VSCode or Microsoftu podobně jako Atom funguje out-of-the-box (nemusí se nic nastavovat).
+Pokud ale chceš v editoru psát Python kód (nebo obecně každý jazyk) nainstaluj si rozšíření pro daný jazyk. 
+
+## Jak na to:
+
+1. Otevři si VSCode editor.
+2. V sloupci úpně na levo(šedý) klikni na ikonku rozšíření. Ve výchozí pozici
+třetí od shora. Vypadá jako čtverec, který má v sobe ještě jeden.
+3. Do vyhledávacího boxu zadej "Python".
+4. Zvol první výsledek (v rohu je označen hvězdičkou).
+5. V nově otevřeném okně klikni na modré tlačítko "Install".
+6. Vyčkej, než se rozšíření nainstaluje. Editor se restartuje a zobrazí se ti okno nápovědy rozšíření.
+
+## Spuštění kódu
+
+V editoru nenajdeš tlačtko spusit kód. Pokud tak chceš učinit naistaluj si toto [Rozšíření](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner).
+Není ale potřeba. Spustit kód můžeš i přes příkaz v terminálu. V horním menu terminálu zvol view, terminal.
+V otevřeném terminálu napiš příkaz: python <název souboru> a stiskni klávesu ENTER. Kód ti nyní běží v terminálu editoru.
```

Comment:
The terminal is explained later.
The `python` command doesn't work on all computers (unless a virtualenv is activated).

Reply:
> Není ale potřeba

Please, do not say this "you may or may not". Either do it fully or don't even mention it. Students have a lot of things to think about already.